### PR TITLE
Drop compiler_blacklist_versions.  Availability of threads.h is deter…

### PR DESCRIPTION
**Pull request by JanWielemaker**
…mined

by CMake and acted upon.

As discussed in https://github.com/macports/macports-ports/pull/16907#discussion_r1041650332

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS x.y
Xcode x.y / Command Line Tools x.y.z

###### Verification 
Have you

- [ ] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [ ] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [ ] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [ ] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

**Comment by barracuda156** [comment]
I guess you rather mention the port in the title. Otherwise it given an impression you wanna drop the PG itself.

File: lang/swi-prolog-devel/Portfile [hunk 1]
```diff
@@ -2,7 +2,6 @@
 
 PortSystem          1.0
 PortGroup           cmake 1.1
-PortGroup           compiler_blacklist_versions 1.0
```
Comment (reneeotten) [inline lang/swi-prolog-devel/Portfile:5]: that is okay, but then you will need remove the lines (38-39 or so) where you do the compiler.blacklist stuff as well.

**Review by reneeotten — CHANGES_REQUESTED** [review]

**Comment by pmetzger** [comment]
The commit message here is in the wrong format; that needs to be fixed. It has to start "swi-prolog-devel:" and probably needs to be turned into multiple lines.

**Comment by JanWielemaker** [comment]
Ok.   I see I'll drop this PR.    As the next release is ready, should I first make a PR for this, make a PR to also do the release update (with one or two commits?)

**Comment by reneeotten** [comment]
Thanks @JanWielemaker - you can do it in the same PR/commit.

**Pull request closed without merging**